ui/sv_panels.py

```diff
@@ -177,14 +174,16 @@ def draw(self, context):
                     for item in tree.Sv3DProps:
                         no = item.node_name
                         ver = item.prop_name
+                        # BUG: item could be in tree.Sv3DProps for some reason, even if it is deleted from tree.nodes!
```

Comment:
good point, i think we should pop deleted node props from sv3DProps in the tree update handler, to make sure we don't need to add code like this. This will be fixed a different way in svb2.8.